docs/pipelines/access-control/index.md

```diff
@@ -0,0 +1,29 @@
+# Access control
+
+Gruntwork Pipelines grants permissions by defining three [GitHub Teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), which should map to three separate personas in your organization. Each team and its permissions are designed to grant the _least possible permissions_ to each individual (or machine user) in your organization for them to be able to perform changes to your infrastructure.
+
+### GitHub Team: infrastructure-administrators
+
+The `infrastructure-administrators` team is for engineers who likely work on the IaC codebase daily, and  have administrative AWS permissions. These users will also have permission to both read and write Pipelines configuration files, and the GitHub Actions Workflow files that are used to trigger Pipelines.
+
+### GitHub Team: infrastructure-collaborators
+
+The `infrastructure-collaborators` team is for engineers who work on the IaC codebase daily but who _do not_ have administrative permissions in AWS. The users will also have permission to read Pipelines configurations files, but not to write (or modify) them.
+
+### Github Team: ci-code-read-only
+
+ The `ci-code-read-only` team is meant for a single machine user that can read your `infrastructure-live` and `infrastructure-modules` (a repository where you can define custom OpenTF (Terraform) modules for your organization) repos.
+
+### Visualizing all three teams
+
+The diagram below visually illustrates the above teams:
+
+![Gruntwork Pipelines Permissions](/img/pipelines/how-it-works/pipelines_security.png)
```

Comment:
This diagram took me a few looks to understand. The prose focuses on the 3-persona structure. The diagram shows how that fits in, but with a primary organization of the _repos_. I think it would help to more clearly indicate that these two big boxes represent repos, perhaps by just labeling the boxes as such instead of using the purple "blocks" inside them to do so. I'd also include the word repo in the title, rather than deferring to the key, e.g., "infrastructure-live repo."

Reply:
Updated in [3a53435](https://github.com/gruntwork-io/docs/pull/1099/commits/3a53435f46b2c61deef66c860a99ed6971fabda9). Let me know your thoughts.